editor.planx.uk/src/@planx/components/FindProperty/Public/Map.tsx

```diff
@@ -111,54 +125,65 @@ export default function PlotNewAddress(props: PlotNewAddressProps): FCReturn {
 
   return (
     <>
-      <MapContainer environment={environment} size="large">
-        <p style={visuallyHidden}>
-          An interactive map centred on the local authority district, showing
-          the Ordnance Survey basemap. Click to place a point representing your
-          proposed site location.
-        </p>
-        {/* @ts-ignore */}
-        <my-map
-          id="plot-new-address-map"
-          ariaLabelOlFixedOverlay="An interactive map for providing your site location"
-          data-testid="map-web-component"
-          zoom={14}
-          drawMode
-          drawType="Point"
-          geojsonData={JSON.stringify(props?.boundary)}
-          geojsonColor="#efefef"
-          geojsonBuffer={10}
-          resetControlImage="trash"
-          showScale
-          showNorthArrow
-          osProxyEndpoint={`${process.env.REACT_APP_API_URL}/proxy/ordnance-survey`}
-          clipGeojsonData={JSON.stringify(boundaryBBox)}
-          collapseAttributions={window.innerWidth < 500 ? true : undefined}
-        />
-        <MapFooter>
+      <ErrorWrapper error={props.mapValidationError} id="plot-new-address-map">
+        <MapContainer environment={environment} size="large">
+          <p style={visuallyHidden}>
+            An interactive map centred on the local authority district, showing
+            the Ordnance Survey basemap. Click to place a point representing
+            your proposed site location.
+          </p>
+          {/* @ts-ignore */}
+          <my-map
+            id="plot-new-address-map"
+            ariaLabelOlFixedOverlay="An interactive map for providing your site location"
+            data-testid="map-web-component"
+            zoom={14}
+            drawMode
+            drawType="Point"
+            drawGeojsonData={
+              coordinates &&
+              JSON.stringify({
+                type: "Feature",
+                geometry: {
+                  type: "Point",
+                  coordinates: [coordinates?.longitude, coordinates?.latitude],
+                },
+                properties: {},
+              })
+            }
```

Comment:
Our current implementation  was actually failing to properly display the new address point when going "back" - now it's fixed to set `drawGeojsonData` here. 

We don't show a visual border of the team boundary, so we also no longer need to set `geojsonData`; `clipGeojsonData` is sufficient !